### PR TITLE
Describe default certificates on Umbraco Cloud

### PR DESCRIPTION
**Pull request by pijemcolu**
The PR describes the certificate for default umbraco.io domains - *umbraco.io. The PR also describes a default Let's Encrypt certificate for custom domains.

### CODE REVIEW

File: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ Learn more about our recommendations for DNS records in the [Manage Hostnames](.
 
 All new Live sites created on Cloud since version 7.12 will automagically have a permanent redirect (301) from HTTP to HTTPS. This is achieved by a web.config transform called: `Latch.Web.live.xdt.config` - accessible in your git repository. If you'd like to remove the redirect rule (which we and [others](https://www.blog.google/products/chrome/milestone-chrome-security-marking-http-not-secure/) strongly discourage) you'll need to remove the file `Latch.Web.live.xdt.config` from projects repository and push the change to Cloud.
 
+## Default TLS Certificates
+
+*.umbraco.io domains will serve a default wildcard certificate with a common name *.umbraco.io.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md]: I think starting the section with `*` made the whole thing appear in italics 🤔 

```suggestion
All `*.umbraco.io` domains will serve a default wildcard certificate with a common name `*.umbraco.io`.
```

File: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md [hunk 2]
```diff
@@ -19,6 +19,12 @@ Learn more about our recommendations for DNS records in the [Manage Hostnames](.
...
+Custom domains will be automatically secured by a Let's Encrypt certificate.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md]:
```suggestion
Custom domains will automatically be secured by a Let's Encrypt certificate.

```

File: Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md [hunk 3]
```diff
@@ -19,6 +19,12 @@ Learn more about our recommendations for DNS records in the [Manage Hostnames](.
...
+You can always upload a custom certificate to overwrite the default certificates provided for the domains.
```
Comment (sofietoft) [inline Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md]:
```suggestion
You can always [upload a custom certificate](../../Manage-Hostnames/Security-Certificates/) to overwrite the default certificates provided for the domains.
```

**Comment by pijemcolu** [comment]
Thanks for the suggestions @sofietoft. I've committed all of them.

**Comment by sofietoft** [comment]
Brilliant! Thanks @pijemcolu 💪 

I'll get this merged right away.

**Pull request merged**